js/main.js

```diff
@@ -1,6 +1,6 @@
 ///////////Global vars/////////////
 // global website base, set to localhost for testing, use deploy script to change
-var baseUrl = "http://eyebrowse.csail.mit.edu";
+var baseUrl = "http://localhost:8000";
```

Comment:
make sure to use `make extension` before deploying so this points to the prod server.